src/main/java/by/custom/utilcalculator/domain/FileUserProgressStorage.java

```diff
@@ -0,0 +1,68 @@
+package by.custom.utilcalculator.domain;
+
+import by.custom.utilcalculator.exception.ReadingUserProgressFromFileException;
+import by.custom.utilcalculator.exception.UserFileNotFoundException;
+
+import java.io.*;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+public class FileUserProgressStorage implements IUserProgressStorage {
+    private final String databaseEnvVariable = System.getenv("CHATBOT_DATABASE_STORAGE");
```

Comment:
I think it corresponds to trues more.
```suggestion
    private final String databaseEnvVariable = System.getenv("CHATBOT_FILE_STORAGE");
```

Reply:
Yes, fixed